src/main/resources/static/js/ButtonDisabler.js

```diff
@@ -0,0 +1,10 @@
+document.addEventListener("DOMContentLoaded", function () {
+    const examNumberInput = document.getElementById("examNumber");
+    const submitButton = document.getElementById("submitButton");
+    function updateButtonState() {
+        const examNumber = examNumberInput.value;
+        submitButton.disabled = examNumber === "";
```

Comment:
Use `.trim()` to account for whitespaces

Reply:
added .trim